src/db.js

```
@@ -22,11 +22,11 @@ const {
 
 const sequelize = new Sequelize({
   database: POSTGRES_DB || 'postgres',
-  username: POSTGRES_USER || 'postgres',
+  username: POSTGRES_USER || 's-stanishevska',
   host: POSTGRES_HOST || 'localhost',
   dialect: 'postgres',
   port: POSTGRES_PORT || 5432,
-  password: POSTGRES_PASSWORD || '123',
+  password: POSTGRES_PASSWORD || '',
```

Line sets the password fallback to an empty string. Avoid defaulting to `''` because it can hide misconfiguration and is insecure. Prefer leaving it undefined (so Sequelize uses env-provided value) or require it explicitly and document it in `.env.example`. If you must provide a local default, document it clearly rather than embedding it in code.